src/lib.rs

```diff
@@ -374,6 +374,7 @@ mod tests {
 
     #[test]
     /// Tests to make sure the error messages look like we expect.
+    /// TODO: Assumes the standard US English translation of each platform.
```

Comment:
haha, translation issues for error messages is a much larger issues IMO. I think this can be removed and an issue opened instead.

Reply:
Sure, I can do that. Something like "change the `sanity_errors()` test to check each individual field and not the `Display` rendering"?

Reply:
I think the best course of action is to:
- remove this line
- open an i18n ticket. This is definitely out of scope for the current library but if someone wanted to address it I would definitely accept a PR.